poetry.lock

```diff
@@ -5154,20 +5136,6 @@ files = [
 [package.extras]
 test = ["pytest (>=6.0.0)", "setuptools (>=65)"]
 
-[[package]]
```

Comment:
Highlighting this one -- It has been removed as a side effect of `poetry remove loguru`

---

How should `poetry.lock` be managed? Should it be generated by some CICD tool? Or maintained by core dev of this project?

Reply:
I think we should ignore for the most part, it's just for consistent builds